webpack.config.js

```diff
@@ -18,14 +18,6 @@ module.exports = {
   module: {
     loaders: [
       {
-        test: /\.js$/,
-        // exclude: /(node_modules)/,
-        loader: 'babel',
```

Comment:
I see. I'd still recommend keeping this, and pointing webpack at the `src` directory because
* It removes a dependency on the state of the dev environment
* Everyone does it this way (I know, I know...)